src/admin/content-block/components/wrappers/MediaGridWrapper/MediaGridWrapper.tsx

```diff
@@ -56,6 +74,73 @@ const MediaGridWrapper: FunctionComponent<MediaGridWrapperProps> = ({
 		[history]
 	);
 
+	const mapSearchData = useCallback(
+		(searchData: Avo.Search.Search): MediaListItem[] => {
+			if (searchData.results.length > 0) {
+				return searchData.results.map(searchItem => {
+					const category = toEnglishContentType(searchItem.administrative_type);
+					const isItem = ['audio', 'video'].includes(category);
+					const duration = formatDurationHoursMinutesSeconds(
+						get(searchItem, 'duration_seconds', 0)
+					);
+
+					return {
+						category,
+						metadata: [
+							{ icon: 'eye', label: String(searchItem.views_count) },
+							{ label: formatDate(searchItem.dcterms_issued) },
+						],
+						navigate: () =>
+							navigateToContentType(
+								{
+									type: isItem
+										? 'ITEM'
+										: (category.toUpperCase() as ContentPickerType),
+									value: searchItem.external_id,
+								},
```

Comment:
mss is het handiger om de navigateToContentType te updaten zodat audio en video ook geinterpreteerd kunnen worden, dan moet je die checks niet meer doen hier.

eg:
```typescript
export const navigateToContentType = (action: ButtonAction, history: History) => {
	if (action) {
		const { type, value } = action;

		switch (type as Avo.Core.ContentPickerType | 'VIDEO' | 'AUDIO') {
			case 'INTERNAL_LINK':
			case 'CONTENT_PAGE':
				history.push(value as string);
				break;
			case 'COLLECTION':
				navigate(history, APP_PATH.COLLECTION_DETAIL.route, {
					id: value as string,
				});
				break;
			case 'ITEM':
			case 'VIDEO':
			case 'AUDIO':
				navigate(history, APP_PATH.ITEM_DETAIL.route, {
					id: value,
				});
				break;
			case 'BUNDLE':
				navigate(history, BUNDLE_PATH.BUNDLE_DETAIL, {
					id: value,
				});
				break;
			case 'EXTERNAL_LINK':
				window.location.href = value as string;
				break;
			default:
				break;
		}
	}
};
```

Reply:
Yes maar dan moet ik wel changes doen aan de ButtonAction die in de comp. lib. zit 🙈